doc/contributing/advocacy-ambassador-program.md

```diff
@@ -165,3 +165,46 @@ Some of the things to highlight include:
 #### Project contacts
 
 * @mhdawson
+
+### How things get done in the Node.js project
+
+#### Goal
+
+Help people understand that no people are paid to answer their issues or
+implement their pull requests. Things get done based on what volunteers work on
+and the best way to get something fixed/changed is to submit a Pull request.
+
+Some of the things to highlight include:
+
+* Nobody is paid specifically to answer issues or implement features.
+* No company owns/supports Node.js. Most contributions are from individuals
+  as opposed to organizations.
```

Comment:
Perhaps worth expanding here: Even if a company does pay their employee to contribute to Node.js, all contributions are individual. Meaning, that once an individual receives a commit bit, that belongs to the individual and not the company. If they change employers their status does not change and granting the commit bit to the individual grants no rights in the project to the company. Just not sure how to say that succinctly.

Reply:
@jasnell, thanks for the suggestion, added some words, let me know if that is what you had in mind.